--- origsrc/rxvt-2.7.10/Makefile.in	2003-03-06 20:24:32.000000000 -0600
+++ src/rxvt-2.7.10/Makefile.in	2016-03-25 16:00:43.430611100 -0500
@@ -25,7 +25,6 @@ MKDIR = $(srcdir)/autoconf/mkinstalldirs
 #-------------------------------------------------------------------------
 
 all allbin alldoc tags:
-	@if test x@host_os@ = xcygwin; then (cd W11; ${MAKE} $@ || exit 1); fi
 	@for I in ${subdirs}; do (cd $$I; ${MAKE} $@ || exit 1); done
 
 realclean: clean
@@ -34,7 +33,6 @@ realclean: clean
 clean:
 	$(RMF) *~ config.cache
 	$(RMF) -r autom4te.cache
-	@if test x@host_os@ = xcygwin; then (cd W11; ${MAKE} $@ || exit 1); fi
 	@for I in ${subdirs}; do (cd $$I; ${MAKE} $@ || exit 1); done
 
 #
@@ -70,7 +68,6 @@ installdirs:
 	$(MKDIR) $(DESTDIR)$(mandir)
 
 install: installdirs
-	@if test x@host_os@ = xcygwin; then (cd W11; ${MAKE} $@ || exit 1); fi
 	@for I in $(subdirs); do (cd $$I; $(MAKE) DESTDIR=$(DESTDIR) $@ || exit 1); done
 
 Makefiles:
